teamcity/resource_build_trigger_schedule.go

```diff
@@ -65,6 +66,44 @@ func resourceBuildTriggerSchedule() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Optional: true,
 			},
+			"cron_schedule": {
+				Type:     schema.TypeList,
```

Comment:
Just to confirm, is this the typical way to add a struct-style configuration (single-element list)? Are there any similar examples of this pattern?

Reply:
I mostly based this structure off of the way `settings` in `resource_build_configs` are handled. Code [here](https://github.com/yext/terraform-provider-teamcity/blob/yext/teamcity/resource_build_config.go#L183). A scema.TypeMap didn't seem to work with this, since we wanted to have specific keys, and would then have to add validation for those keys.

Reply:
Thanks for clarifying! LGTM